build-parent/pom.xml

```diff
@@ -495,6 +502,14 @@
         <formatter.goal>validate</formatter.goal>
         <impsort.goal>check</impsort.goal>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
```

Comment:
Are you sure it is necessary to declare this dependency also here? As far as I see it seems to be used only by the `measure-code-coverage` profile